### PR TITLE
Incus 0.1

### PR DESCRIPTION
**Pull request by juippis**
incus is a fork from Canonical's LXD.

### CODE REVIEW

File: app-containers/incus/incus-0.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+WARNING_VHOST_VSOCK="CONFIG_VHOST_VSOCK is required for virtual machines."
+
+# Go magic.
+QA_PREBUILT="/usr/bin/incus
+	/usr/bin/lxc-to-incus
+	/usr/bin/lxd-to-incus
+	/usr/bin/incus-agent
+	/usr/bin/incus-benchmark
+	/usr/bin/incus-migrate
+	/usr/sbin/incusd"
```
Comment (sevz17) [inline app-containers/incus/incus-0.1.ebuild:75]: Aren't these supposed to be relative?

Comment (juippis) [reply app-containers/incus/incus-0.1.ebuild:75]: `${D}` should be prefixed here either way. I'm pretty sure this works, too.

File: dev-libs/cowsql/cowsql-0_pre20230921.ebuild [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+RDEPEND="dev-db/sqlite:3
+	dev-libs/libuv:=
+	>=dev-libs/raft-0.17.1:="
+DEPEND="${RDEPEND}
+	test? ( dev-libs/raft[lz4,test] )"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=( "${FILESDIR}"/dqlite-1.12.0-disable-werror.patch )
+
+S="${WORKDIR}/cowsql-${MY_COMMIT}"
```
Comment (mm1ke) [inline dev-libs/cowsql/cowsql-0_pre20230921.ebuild:29]: should be below `SRC_URI`

**Comment by MatthewGentoo** [comment]
Does Incus replace LXD/will LXD be dropped from Gentoo?

**Pull request closed without merging**